server/controllers/prisonerLocationDetailsController.ts

```diff
@@ -90,7 +63,7 @@ export default class PrisonerLocationDetailsController {
         page: Page.PrisonerCellHistory,
       })
 
-      // TODO CDPS-373: Move into PrisonerLocationDetailsService
+      // TODO CDPS-373: Move into PrisonerLocationDetailsService?
```

Comment:
At risk of opening a can of worms, do we have an agreement with our code structure?

* We have a mix of router definitions defined in `routes/index.ts` and in separate files
* We then have controllers, with varying degrees of logic implemented in the controller, ranging from all of it to almost none.
* We have some services that are aligned to a domain - i.e. `appointmentService.ts`
* We have some services that are purely there to deliver the data for a single page - i.e. `overviewPageService.ts`

How far should a service go? Should it be packaging up all the data needed for a page or should that be the controller's job?

Reply:
@brightonsbox Yeah, it's all evolved as more functionality has been added.

My view is...

Having separate `routes` makes sense where there are multiple associated pages, like case notes (list, add, update), etc. If it's just single pages, not much point moving out of `index`

Controllers should handle view related logic and orchestrate calls to multiple services (where required)

Services should be decoupled from the view, handle "business logic" and orchestrate calls to multiple APIs (where required)

Should probably have a team discussion on this and write an ADR

Reply:
We should def discuss. Def not a fan of the Page services. The controllers should orchestrate calls for data from the services which are domain agnostic. Especially dislike it when they return data shapes for nunjucks. 

Believe that is pretty much what David said anyway!